src/graph/fragments/log.ts

```diff
@@ -47,7 +47,6 @@ export const QueryServiceLogs = {
           endpointId
           traceId
           timestamp
-          isError
           statusCode
```

Comment:
I think this should be removed too based on today's tag system. @arugal @kezhenxu94